config/crd/v1/bases/velero.io_backups.yaml

```diff
@@ -37,6 +37,11 @@ spec:
           spec:
             description: BackupSpec defines the specification for a Velero backup.
             properties:
+              csiSnapshotTimeout:
```

Comment:
could you also update the spec doc [here](https://velero.io/docs/v1.9/api-types/)?

Reply:
Added.